langwatch/src/pages/api/triggers.ts

```diff
@@ -0,0 +1,114 @@
+import { type NextApiRequest, type NextApiResponse } from "next";
+
+import { type Trigger, TriggerAction } from "@prisma/client";
+import { getAllForProject } from "~/server/api/routers/traces";
+import { prisma } from "../../server/db";
+
+import { sendTriggerEmail } from "~/server/mailer/triggerEmail";
+import { getLatestUpdatedAt } from "./utils";
+
+interface ActionParams {
+  members?: string[] | null;
+  dataset?: string | null;
+}
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method !== "POST") {
+    return res.status(405).end(); // Only accept POST requests
+  }
+
+  const triggers = await prisma.trigger.findMany({
+    where: {
+      active: true,
+    },
+  });
```

Comment:
I see maybe this is why you had to disable the protection right, well, what if you query all projectIds first and pass it here as an explicit argument as an array? This way later we can even filter our projects that are disabled or something